### DISCUSSION TITLE
Why osu! sound so delay when playing on some pc?

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#14854] [Q&A] [answered]
So I have asus laptop with touchscreen and I install osu! and I see the sound of osu! is REALLY delay even though I use compaitability audio.  
is there problem with client or my laptop?

**Comment by peppy** [comment, marked as answer]
You are opening a discussion against osu!(lazer). Judging from your comment you are not using lazer.

For troubleshooting see https://osu.ppy.sh/help/wiki/Performance_Troubleshooting.
